app/src/main/java/org/bspb/smartbirds/pro/ui/map/GoogleMapProvider.java

```diff
@@ -426,7 +468,17 @@ private void displayGeometry(KmlGeometry geometry, Style style) {
         } else if (geometry instanceof KmlMultiGeometry) {
             KmlMultiGeometry multiGeometry = (KmlMultiGeometry) geometry;
             for (KmlGeometry subGeometry : multiGeometry.mItems) {
-                displayGeometry(subGeometry, style);
+                displayGeometry(subGeometry, style, null);
+            }
+        } else if (geometry instanceof KmlPoint) {
+            KmlPoint kmlPoint = (KmlPoint) geometry;
+            ArrayList<GeoPoint> geopoints = kmlPoint.mCoordinates;
+            for (GeoPoint point : geopoints) {
+                MarkerOptions markerOptions = new MarkerOptions()
+                        .position(new LatLng(point.getLatitude(), point.getLongitude()))
+                        .icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_BLUE))
+                        .title(name);
+                mMap.addMarker(markerOptions);
```

Comment:
Is this needed for the stork local project? It seems to me storks are handled separately inside drawLocalProjects and this makes me think would it conflict with each other.

Reply:
This allows drawing points from area.kml file.
I won't conflict with local projects.